src/content/docs/zh-cn/guides/fonts.mdx

```diff
@@ -80,39 +71,59 @@ Astro 支持所有常见的在网站中添加自定义字体的方法。本指
     </PackageManagerTabs>
 
     :::tip
-    你可以在 Fontsource 上每个字体页面的“快速安装”部分找到正确的包名称。包名总是以 `@fontsource/` 字体名称开头。
+    你可以在 Fontsource 网站上每个字体页面的“快速安装”部分找到正确的包名称。包名总是以 `@fontsource/` 字体名称开头。
     :::
 
-3. 在要使用字体的布局或组件中导入字体包。通常，需要在通用布局组件中执行此操作，以确保该字体在你的站点中可用。
+1. 在要使用字体的布局或组件中导入字体包。通常，需要在通用布局组件中执行此操作，以确保该字体在你的网站上可用。
 
-    导入后 Astro 并添加 `@font-face` 以设置字体所需的必要属性。
+    导入（import）将自动添加 `@font-face` 以设置字体所需的必要属性。
 
-    ```astro
+    ```astro title="src/layouts/BaseLayout.astro"
     ---
-    // src/layouts/BaseLayout.astro
     import '@fontsource/twinkle-star';
     ---
     ```
 
-4. 使用该 `font-family` 字体的 Fontsource 页面上显示的。这将适用于可以在 Astro 项目中编写 CSS 的任何地方。
+1. 使用字体名称作为 `font-family` 值，正如 Fontsource 网站中的例子所示。这适用于任何可以在 Astro 项目中编写 CSS 的地方。
 
     ```css
     h1 {
       font-family: "Twinkle Star", cursive;
     }
     ```
 
-## 更多资源
+## 在 Tailwind 中注册字体
+
+如果你使用的是 [Tailwind 集成](/zh-cn/guides/integrations-guide/tailwind/)，则可以使用此页面的前述方法之一来安装字体，并进行一些修改。你也可以使用[`@font-face` 语句注册本地字体](#使用本地字体文件)，或使用 [Fontsource 的 `import` 策略](#使用字体资源包)去安装你的字体。
+
+要在 Tailwind 中注册你的字体：
 
-### 使用 Tailwind 添加字体
+1. 请遵循上述任一指南，但跳过添加 `font-family` 到 CSS 的最后一步。
+1. 将字体名称添加到 `tailwind.config.cjs`。
 
-如果您使用的是 [Tailwind 集成](/zh-cn/guides/integrations-guide/tailwind/)，您可以直接为本地字体添加 `@font-face`，或使用 Fontsource 的 `import` 策略来注册您的字体，然后按照 Tailwind 的文档添加自定义字体。
+    此示例将添加 `InterVariable` 和 `Inter` 到 sans-serif 字体堆栈，并使用 Tailwind CSS 的默认备用字体。
 
-### 了解 Web 字体如何运作
+    ```js title="tailwind.config.cjs" ins={1,7-9}
+    const defaultTheme = require("tailwindcss/defaultTheme");
 
-[MDN的 Web 字体指南](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Web_fonts) 介绍了这个话题。
+    module.exports = {
+      // ...
+      theme: {
+        extend: {
+          fontFamily: {
+            sans: ["InterVariable", "Inter", ...defaultTheme.fontFamily.sans],
+          },
+        },
+      },
+      // ...
+    };
+    ```
+
+    现在，项目中的所有 sans-serif 文本（Tailwind 的默认设置）都将使用你选择的字体，并且该类 `font-sans` 也将应用 Inter 字体。
 
-### 为你的字体生成 CSS
+有关的更多信息，请参阅[Tailwind 中关于添加自定义字体系列的文档](https://tailwindcss.com/docs/font-family#using-custom-values)。
 
-[Font Squirrel 的 Web 字体生成器](https://www.fontsquirrel.com/tools/webfont-generator)会帮你准备字体。
+## 更多资源
+[在 MDN 的网站字体指南](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Web_fonts) 中了解网站字体的工作原理。
+[使用 Font Squirrel 的 Webfont Generator](https://www.fontsquirrel.com/tools/webfont-generator)为你的字体生成 CSS。
```

Comment:
It looks like you are missing something here?

https://github.com/withastro/docs/blob/5a14ecca23cb57c7156bb2e11097b648aee84eb3/src/content/docs/en/guides/fonts.mdx#L128-L129

```suggestion
- [在 MDN 的网站字体指南](https://developer.mozilla.org/en-US/docs/Learn/CSS/Styling_text/Web_fonts) 中了解网站字体的工作原理。
- [使用 Font Squirrel 的 Webfont Generator](https://www.fontsquirrel.com/tools/webfont-generator)为你的字体生成 CSS。
```